app/views/operations/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="new-transactio-container">
+<header>
+<%= link_to group_operations_path(@group) do %>
+  <%= fa_icon "arrow-left" %> 
+<% end %>
+<h2>Add a New Transaction</h2>
+<%= fa_icon "search", class: "header-icons" %>
+</header>
+
+<%= form_for([@group, @operation]) do |f| %>
+
+  <div class="form-group">
+    <%= f.text_field :name, placeholder: "Transaction Name", required: true, class: 'form-control' %>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :amount, "Amount" %>
+    <%= f.number_field :amount, required: true, step: "0.01", class: 'form-control' %>
+  </div>
+
+  <button type="submit" class="add-category link-new-category">SAVE</button>
+
+<% end %>
```

Comment:
- [ ] As stated in the project requirement For "Add a new transaction" page, The user fills out a form to create a new **transaction with: name (mandatory), amount (mandatory) and categories (mandatory at least one)**. But Your Design Misses  categories (mandatory at least one) So kindly Adjust the design as mentioned in the project requirement. 
![image](https://github.com/bohaz/Budget-app/assets/93598189/cddfb33f-5261-4808-af17-ed922993f934)

Reply:
## Thanks, @Abenezer-Tilahun. 
### I appreciate your professional review. 
I understand perfectly the required change, however, how you can see the new transaction is created from a selected category. Is it necessary to select the same category again? I kindly hope you respond to this message. 

Greetings!